tungsten_ci_utils/mirror_maven_repo/deploy.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+
+repoUrl='http://ci-nexus.englab.juniper.net/repository/vmware-releases/'
+VERSION='7.3.0'
+
+# Deploy artifacts composed of a pom, jar, sources and javadocs jar
+
+cd 4
+for file in `ls -A1 | grep pom`; do prefix=${file%%-${VERSION}*}; mvn deploy:deploy-file -DpomFile=${prefix}-${VERSION}.pom \
```

Comment:
Use `$()` instead of `` ` `` for command substitution.

Reply:
Done.